### DISCUSSION TITLE
Github Octernships

### DISCUSSION THREAD

**Discussion by Aryankpoor** [community/community#134567] [Other Feature Feedback, Questions, & Ideas]
### Github Octernships

So the last cycle of github octernships ended in january 2024. When will the next cycle for applications open? Also how long do the octenships usually last?

**Comment by ebndev** [comment]
👋 Hi @Aryankpoor - Thanks for your interest in GitHub Octernships. The program is currently on pause and we are not initiating new rounds or accepting new partners at this moment. We appreciate your understanding and support.

Please view [this original comment](https://github.com/orgs/community/discussions/113185#discussioncomment-9356088) if you have any additional questions!
